.github/workflows/main.yml

```diff
@@ -568,9 +568,41 @@ jobs:
         run: |
           set -x
           sudo rm -rf "/usr/local/share/boost" "$AGENT_TOOLSDIRECTORY" "/usr/local/lib/android" "/usr/share/dotnet"
-          docker run --rm -d --name indypool --network host $DOCKER_IMG_NAME_POOL
-          docker run --rm -d --name postgres --network host -e POSTGRES_PASSWORD=mysecretpassword postgres:12.1
+          docker run --rm -d --name indypool  --network host $DOCKER_IMG_NAME_POOL
+          docker run --rm -d --name mysql     --network host -e MYSQL_ROOT_PASSWORD=mysecretpassword mysql:5.7.35
+
+      - name: Create Wallet DB schema, run migrations
+        run: |
+          set -x
+          docker run --rm --name vcxagencynode_migrations_wallet \
+                         --network host \
+                          -e MYSQL_DATABASE=agency_wallets \
+                          -e MYSQL_USER=root \
+                          -e MYSQL_PASSWORD=mysecretpassword \
+                          -e MYSQL_HOST=localhost \
+                          -e MYSQL_PORT=3306 \
+                         "$DOCKER_IMG_NAME_AGENCY" \
+                         bash -c "cd /home/node/dbutils && yarn install && yarn run schema:migrate:wallet"
+
+      - name: Create Application DB schema, run migrations
+        run: |
+          set -x
+          docker run --rm --name vcxagencynode_migrations_data \
+                         --network host \
+                          -e MYSQL_DATABASE=agency_application \
+                          -e MYSQL_USER=root \
+                          -e MYSQL_PASSWORD=mysecretpassword \
+                          -e MYSQL_HOST=localhost \
+                          -e MYSQL_PORT=3306 \
+                         "$DOCKER_IMG_NAME_AGENCY" \
+                         bash -c "cd /home/node/dbutils && yarn install && yarn run schema:migrate:app"
```

Comment:
These two steps run a container, perform a migration and then proceed to delete it, which does not seem very useful. What am I missing?